oppijanumerorekisteri-service/pom.xml

```diff
@@ -250,5 +250,36 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <plugins>
+                    <!-- Run Flyway DB migrations to confirm they work -->
+                    <plugin>
+                        <groupId>org.flywaydb</groupId>
+                        <artifactId>flyway-maven-plugin</artifactId>
+                        <version>6.0.6</version>
```

Comment:
Pitäskö ottaa versionumero pois niin käyttää samaa versiota kuin sovelluskin (eli spring bootin tarjoamaa versiota)?

Reply:
Pitäis.